_posts/2022-01-27-partner-hidora.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Partner Highlight: Enjoy OpenSearch one click installation with Hidora"
+authors:
+  - piellick 
+date: 2022-01-27
+categories:
+  - partners
+twittercard:
```

Comment:
Before this line add:
```
excerpt: Hidora is an expert in providing custom solutions for all sizes of businesses. Hidora Cloud Hosting Solutions are just what you need if you want an efficient, secure and scalable cloud hosting solution that is tailored to your needs.
```

Reply:
hello @stockholmux, before wich line ?

i change like that ?  

---
layout: post
title: "Partner Highlight: Enjoy OpenSearch one click installation with Hidora"
authors:
  - pprost 
date: 2022-01-27
categories:
  - partners

excerpt: "Hidora is an expert in providing custom solutions for all sizes of businesses. Hidora Cloud Hosting Solutions are just what you need if you want an efficient, secure and scalable cloud hosting solution that is tailored to your needs. We offer different plans that are scalable according to your specific requirements so you can find one that suits you best. Our PaaS help developers to host their applications in few minutes with DevOps philosophy."
---